### ISSUE TITLE
Using pygame as a dependency inside a virtualenv crashes in segmentation fault - pygame 1.9.4

### ISSUE THREAD

**Issue by iuliaav** [pygame/pygame#479]
I'm trying to run a package which uses pygame as a dependency. And I'm trying to run it from inside a virtual environment.

```
pygame 1.9.4
Hello from the pygame community. https://www.pygame.org/contribute.html
pygame 1.9.4
Hello from the pygame community. https://www.pygame.org/contribute.html
ERROR: pygame.macosx.Video_AutoInit() call FAILED
ERROR: pygame.macosx.Video_AutoInit() call FAILED
Fatal Python error: (pygame parachute) Segmentation Fault
Traceback (most recent call last):
  File "path/to/my/project/venv/lib/python3.6/site-packages/pygame/macosx.py", line 11, in Video_AutoInit
    if 'Darwin' in platform.platform():
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/platform.py", line 1329, in platform
    system, node, release, version, machine, processor = uname()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/platform.py", line 1001, in uname
    processor = _syscmd_uname('-p', '')
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/platform.py", line 755, in _syscmd_uname
    f = os.popen('uname %s 2> %s' % (option, DEV_NULL))
  File "/path/to/my/project/venv/lib/python3.6/os.py", line 978, in popen
    bufsize=buffering)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 1260, in _execute_child
    restore_signals, start_new_session, preexec_fn)
TypeError: fork_exec() argument 4 must be tuple, not list
Abort trap: 6
```

This problem doesn't happen when I install the required dependencies globally.

**Comment by iuliaav** [comment]
Hmmm, I'm having a hard time doing that, because pygame is called from within another package I'm using. I've tried modifying the runner file of the package I mentioned to use "pythonw" instead of "python", but it has the same effect. 

Another funny thing I've found is this: if I install pygame globally, but my package in a virtual environment, the error above doesn't reproduce anymore. So I'm guessing pygame just doesn't like being inside a virtualenv?
